Jenkinsfile

```diff
@@ -137,8 +136,8 @@ def executeTests() {
     printenv | sort
 
     mvn -B -V ${INTEGRATION_TESTS_FILTER_ARGUMENT} -T 1 verify \
-      -Ptest-jdk-${TEST_JAVA_VERSION} \
-      -DtestJavaHome=${TEST_JAVA_HOME} \
+      -Ptest-jdk-'''+testJavaVersion+''' \
+      -DtestJavaHome='''+testJavaHome+''' \
```

Comment:
We can't use string interpolation here since the ${} vals are intended to be passed through literally to the shell (so that the shell can do variable subsitution based on env vars).  So instead we just add whatever we got for the two vars above directly into the multi-line string here and then continue on with a new multi-line string.